docs/source/learners/retrieval.rst

```diff
@@ -1,2 +1,63 @@
 Retrieval Models
 =================
+Retriever-only learners use embedding-based retrieval models
```

Comment:
Please, explain a bit the methodology, maybe adding a single paragraph would be great!